recipes/cppcommon/all/patches/00001-update-cmakelists.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index dcd57785b..7e3a1c756 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -17,49 +17,6 @@ if(DOXYGEN_FOUND)
+   endif()
+ endif()
+ 
+-# CMake module path
+-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+-
+-# Compiler features
+-include(SetCompilerFeatures)
+-include(SetCompilerWarnings)
+-include(SetPlatformFeatures)
+-include(SystemInformation)
+-
+-# External packages
+-find_package(Threads REQUIRED)
+-if(UNIX AND NOT APPLE AND NOT MSYS)
+-  find_package(LibBFD)
+-  find_package(LibDL)
+-  find_package(LibRT)
+-  find_package(LibUUID)
+-endif()
+-if(WIN32 OR MSYS)
+-  find_package(DbgHelp)
+-  find_package(RPC)
+-  find_package(Userenv)
```

Comment:
I see some of the system deps are listed in the recipe, can you confirm the rest are not required? ie its not linking against the build image ones

Reply:
Estava fazendo alguns testes com os exemplos e benchmarks da biblioteca pra validar isso...